routes/user.js

```diff
@@ -0,0 +1,13 @@
+const router = require('express').Router();
+const userCtrl = require('../controllers/user');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
const userCtrl = require("../controllers/user");
```